src/pages/index.tsx

```diff
@@ -23,7 +23,7 @@ export const App: React.SFC = () => {
           <Route exact path="/" component={HomePage} />
           <Route path="/target/:targetShortCode" component={TargetPage} />
           <Route path="/search" component={SearchPageRoute} />
-          <Route path="/apps" component={AppsPage} />
+          <Route path="/resources" component={AppsPage} />
```

Comment:
If we're going to rename the route to `/resources`, we should rename the component itself too. It'll be confusing for future devs to remember that the `AppsPage` component renders at the `/resources` route.